### PR TITLE
M2m

### PR DESCRIPTION
**Pull request by jyutzler**

### CODE REVIEW

File: spec/1_related_tables.adoc [hunk 1]
```diff
@@ -72,7 +72,7 @@ The relation defined in the `gpkgext_relations` table shall use id values (prima
 [caption=""]
 .Requirement 2
 ====
-A GeoPackage that complies to this extension SHALL contain a mapping table for each row in the `gpkgext_relations_table`. The name of the table SHALL correspond to the value in the `mapping_table` column. The mapping table MAY contain no rows. Each row that is present SHALL map a row of a feature/attribute/tile table to a row of the attribute table. Two columns SHALL be used to map between the core data table using the `primary_column_id` and the attribute table using the `attribute_column_id`.
+A GeoPackage that complies to this extension SHALL contain a mapping table for each row in the `gpkgext_relations_table`. The name of the table SHALL correspond to the value in the `mapping_table` column. The mapping table MAY contain no rows. Each row that is present SHALL map a row of a feature/attribute/tile table to a row of the attribute table. The two columns SHALL be used to map between the core data table using the `primary_column_id` and the attribute table using the `attribute_column_id`.
```
Comment (dodd5) [inline spec/1_related_tables.adoc]: For consistency, change 'gpkgext_relations_table' to 'gpkgext_relations' table

**Comment by rcasscslt** [comment]
Are we going to need a cardinality column? When we do exports/imports, we will need a way to take the mapping table requirement in external systems iff many-to-many is the cardinality. Is there a quick way to determine a relation's cardinality based on the contents of the mapping table? I'm not sure there is.

**Comment by bradh** [comment]
Jeff: Any objections to merging this?

**Comment by jyutzler** [comment]
@bradh I wanted to give it a couple of days, but no. Merging now!

**Pull request merged**